api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -44,18 +46,27 @@
   (ts).tv_nsec = ((msec) % 1000) * 1000000; \
 } while (0)
 
+/** verify the magic number for ml_single obj */
+#define ML_SINGLE_MAGIC_CHECK(arg) do { \
+  if (arg->magic != ML_SINGLE_MAGIC) { \
+    ml_loge ("The given param, single is invalid."); \
+    return ML_ERROR_INVALID_PARAMETER; \
+  } \
+} while (0)
```

Comment:
It'd be better to define in a common header for both C files.

Reply:
A common header would introduce big changes, irrelevant to this PR. How about a new PR for that?

Reply:
Ok.

Please note that you may place an internal header at /api/capi/src/
or
Let nnstreamer-capi-single.c include tensor_filter_single.h, which minimizes changes.

Reply:
I will add after #1672 is merged